tests/manage/add_metadata_feature/test_cephfs_metadata.py

```diff
@@ -0,0 +1,939 @@
+import pytest
+import logging
+
+from ocs_ci.utility import metadata_utils
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.helpers import helpers
+from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.ocs.resources import pod, pvc
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+    tier3,
+    skipif_ocp_version,
+    skipif_managed_service,
+    skipif_disconnected_cluster,
+    skipif_proxy_cluster,
+    pre_upgrade,
+)
+
+
+log = logging.getLogger(__name__)
+# Error message to look in a command output
+ERRMSG = "Error in command"
+
+
+@tier1
+@skipif_ocs_version(">4.11")
+@skipif_ocp_version(">4.11")
+@skipif_managed_service
+@skipif_disconnected_cluster
+@skipif_proxy_cluster
+@green_squad
+class TestMetadataUnavailable(ManageTest):
```

Comment:
Polarion ID marks are missing from all test cases

Reply:
Have added polarion ids for the tests.